server/http/server.go

```diff
@@ -89,6 +83,19 @@ func (s *Server) HandleSubscriptions() error {
 	return nil
 }
 
+func optionsHandler(h http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == "OPTIONS" {
+			w.Header().Set("Access-Control-Allow-Origin", "*")
+			w.Header().Set("Access-Control-Allow-Methods", "POST,OPTIONS")
+			w.Header().Set("Access-Control-Allow-Headers", "authorization, origin, content-type, accept")
+			w.Header().Set("Allow", "POST,OPTIONS")
```

Comment:
All these header keys are defined in net/http. please use them.

Reply:
same as above, https://golang.org/pkg/net/http doesn't define constants for http header keys

Reply:
I have confused with fasthttp package. But I would define our own const instead of using string literal.